web/ui/react-app/src/pages/targets/Filter.module.css

```diff
@@ -0,0 +1,5 @@
+.btn {
```

Comment:
You realize that by introducing CSS modules as yet another way to do CSS in this project, you just volunteered yourself for converting everything else (that is applicable) to CSS modules as well :P

Reply:
Sure, I can do that. Or I can just make this a single style sheet. What would you prefer? I can do whatever

Reply:
@hooten To be able to judge that better I would first have to learn about CSS modules (eager to do that actually, just not right now for me), how they work, pros/cons, etc.

I'm fine with them being introduced now, but it does mean another way of doing things, and we should understand that we *should* eventually converge on fewer ways.